.gitignore

```diff
@@ -15,6 +15,7 @@ dist
 /doc
 /log
 *.log
+/beakerdsl/.vagrant
```

Comment:
beakerdsl dirs were removed, so this is OBE

Reply:
:ballot_box_with_check:  c058865